charts/nlk/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+appVersion: 0.1.0
+description: NGINX LoadBalancer Kubernetes
```

Comment:
Should this be "NGINX LoadBalancer for Kubernetes" which is the full name? (I am good either way...)